docs/releases.rst

```diff
@@ -4,6 +4,11 @@
     Releases
 ======================
 
+tmt-1.35
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``tmt try`` now supports :ref:`/stories/cli/try/option/epel` option.
```

Comment:
One extra line between the heading and your addition, please. Some parsers might complain about it, although none would probably try to squeeze header and text together.

Reply:
Plus, I would mention the `prepare/feature` plugin as well, something like "backed by `prepare/feature` plugin, https://tmt.readthedocs.io/en/stable/plugins/prepare.html#feature.

Reply:
Added but why does reference on `plugins` doesnt need slash ("/") when others do:
`plugins/prepare/feature` vs `/stories/cli/try/option/epel`

Reply:
I suppose that's because of how the link is constructed, the one in plugins template lacks leading `/`:

```
.. _{{ STORY.name | trim }}:

.. _plugins/{{ STEP }}/{{ PLUGIN_ID | trim }}:
```

`STORY.name` will always start with `/`, `/stories/cli/test/lint` and so on, so the `/` sneaks in. I guess we can fix the plugin template, to make things consistent. Luckily, Sphinx will report all broken links.

Reply:
Submitted in https://github.com/teemtee/tmt/pull/3104